shared/lib/error-utils.js

```diff
@@ -87,3 +107,57 @@ export async function getErrorHtml(errorKey, supportLink, metamaskState) {
     </div>
     `;
 }
+
+function disableDesktop(backgroundConnection) {
+  backgroundConnection.disableDesktopError();
+}
+
+export function downloadDesktopApp() {
+  global.platform.openTab({ url: 'https://metamask.io/' });
```

Comment:
This could be a constant in the new `shared/constants/desktop.ts`.

Reply:
We have an [open task](https://github.com/orgs/MetaMask/projects/37/views/4?filterQuery=link&pane=issue&itemId=20544786) to review all the desktop links. We can tackle it all within that task.